torch/distributions/exponential.py

```diff
@@ -39,3 +40,12 @@ def log_prob(self, value):
 
     def entropy(self):
         return 1.0 - torch.log(self.rate)
+
+    @lazy_property
+    def natural_params(self):
+        V1 = -self.rate.data
+        return (V1, )
+
+    @property
```

Comment:
A property that returns a lambda is just a function :smile:

Reply:
I realized that after I pushed, because this is redundant.